main.go

```diff
@@ -42,6 +42,14 @@ type flagVal struct {
 	OverWrite_flag bool
 	Instance_flag  string
 	Verbose_flag   bool
+	File_flag      bool
+}
+
+// contains local and server paths
+type pathVal struct {
+	RootWorkingDirectoryLocal  string
+	RootWorkingDirectoryServer string
+	StartingPathServer         string
```

Comment:
Yup, a refactor to get rid of it and get `Download` to just take `StartingPathServer` is the plan. `GetDirectoryContext` returned three paths when I began work on this, so I figured the last person to update it had a reason.